app/assets/v2/js/pages/onboard.js

```diff
@@ -226,21 +227,21 @@ keywords.forEach(function(keyword) {
 
 $('#skills #suggested-tags').html(suggested_tags);
 
-if ($('.navbar #navbarDropdown').html()) {
-  var url = '/api/v0.1/profile/' + $('.navbar #navbarDropdown').html().trim() + '/keywords';
+if (document.contxt.github_handle) {
+  var url = `/api/v0.1/profile/${document.contxt.github_handle}/keywords`;
```

Comment:
changed this because menu changed and now $('.navbar #navbarDropdown') wasn't the user gh, was "Products"